### PR TITLE
Edit the manual

### LINKED ISSUE #442: Edit the manual
Read over the manual and update / expand where necessary (filing issues for larger chunks of work)

### PR DESCRIPTION
**Pull request by djmitche**
This is a bit of a WIP, as I'm not sure where to put the stuff in `mozilla-namespaces.md`.  Perhaps in firefox-source-docs?  @tomprince, @callek, opinions?

Fixes #442.

### CODE REVIEW

File: ui/docs/manual/tasks/scopes.md [hunk 1]
```diff
@@ -24,3 +24,7 @@ In fact, several scope sets must be satisfieid to create a task:
  * Scopes corresponding to some components of the task definition, such as
    routes, priorities, and `schedulerId`.
  * The task's `scopes` property
+
+*NOTE*: this means that scopes for a task must often be configured in two places.  For example, to use a scope `secrets:get:org-project-test-creds`, a task associated with a github project would need:
```
Comment (imbstack) [inline ui/docs/manual/tasks/scopes.md]: We should just email this to our users once per day.

File: ui/docs/manual/using/secrets.md [hunk 2]
```diff
@@ -17,8 +17,8 @@ The [Secrets service](/docs/reference/core/secrets) provides a simple way to sto
 JSON-formatted secrets in a secure fashion. Access to secrets can be controlled
 by [scopes](/docs/manual/design/apis/hawk/scopes).
 
-The most common approach is to use the Tools site to create secrets named
-according to the [namespaces document](/docs/manual/design/namespaces), then read
+The most common approach is to use the Taskcluster UI to create secrets named
+according to the [namespaces document](/docs/manual/usin/namespaces), then read
```
Comment (imbstack) [inline ui/docs/manual/using/secrets.md]: `using`?

**Comment by djmitche** [comment]
> Looks good. Minor nit: let's add an h1 tag for pages that have an h2 tag.

I looked through the changed files and fixed this in `ui/docs/manual/design/logs/README.md`.

**Comment by djmitche** [comment]
I moved the mozilla-specific stuff out into ci-configuration and ci-taskgraph in https://bugzilla.mozilla.org/show_bug.cgi?id=1560129.

File: ui/docs/manual/README.md [hunk 3]
```diff
@@ -5,36 +5,22 @@ title: Taskcluster Manual
 
 # Taskcluster Manual
```
Comment (helfi92) [inline ui/docs/manual/README.md]: I think there are two h1's in this file. We should probably have only one.

File: ui/docs/manual/task-execution/worker-pools.md [hunk 4]
```diff
@@ -1,16 +1,25 @@
 ---
-filename: task-execution/worker-types.md
-title: Worker Types
+title: Worker Pools
 order: 30
 ---
 
-There are a few different reasons to create new Taskcluster worker types.
```
Comment (helfi92) [inline ui/docs/manual/task-execution/worker-pools.md]: I think this file is missing an h1.

**Pull request merged**